.github/workflows/cicd.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - run: pip install -U tox poetry
+      - run: pip install tox poetry==1.4.2
```

Comment:
Do we want to pin `tox` version as well?

Reply:
I tried that but there was an incompatibility between the latest version of tox and poetry. Some day soon we'll pin 🤞 .